Mu2eG4/src/ConstructMaterials.cc

```diff
@@ -1557,6 +1557,16 @@ namespace mu2e {
       wires->AddMaterial(findMaterialOrThrow("G4_Au"), 0.06);
     }
 
+    //Carbon steel
+    mat = uniqueMaterialOrThrow( "Steel");
```

Comment:
We have many different materials named Steel.   Please modify the name to indiciate what sort of steel it is (or what it is used for).  Please move this code so that it's in the same place as all of the other steel definitions.

Reply:
Changed in [663d5db](https://github.com/Mu2e/Offline/pull/1001/commits/663d5db9a1656a96b41a01e9447fb703ac37d8d8)